### ISSUE TITLE
Cannot see any cluster details

### ISSUE THREAD

**Issue by makowskimateusz** [kontena/lens#3891]
I facing the issue that when I'm connected to the cluster I'm not able to see anything. 

What I'm I doing wrong? 
![Screenshot 2021-09-24 at 13 04 23](https://user-images.githubusercontent.com/5725006/134664636-aee4fc0f-6c80-468d-a871-b501e52418a9.png)
![Screenshot 2021-09-24 at 13 04 30](https://user-images.githubusercontent.com/5725006/134664640-53e6eb22-bc19-4675-b27a-e9bcbc20ee1c.png)
![Screenshot 2021-09-24 at 13 04 35](https://user-images.githubusercontent.com/5725006/134664641-6a4035cf-c8a9-44cb-b99a-3881628bdcfc.png)

**Comment by Startouf** [comment]
I'm having the same problem on some of my clusters

Lens: 5.2.4-latest.20210923.1
Electron: 12.0.17
Chrome: 89.0.4389.128
Node: 14.16.0
© 2021 Mirantis, Inc.

**Comment by Startouf** [comment]
I can connect to my active context of my .kube/config but I'm having troubles using a different context in my `.kube/config`  or a different file `.kube/config-production`. 

When I copy paste the relevant part of my kubeconfig into a new cluster config "add from kubeconfig" I can successfully connect to this cluster however.

**Comment by robbesettlemint** [comment]
Also facing this issue with one particular cluster. Strange thing is, all my other clusters are perfectly fine (clusters on eks and gks). The cluster that's not working is one on EKS, but other EKS networks work fine. I can't find anything that would differentiate this cluster from the other ones, except that I used this cluster the most.

**Comment by TemplateXu** [comment]
same problem

**Comment by Nokel81** [comment]
If you disconnect and then reconnect to that cluster does the problem still persist?

**Comment by brainstain** [comment]
I am having the same issue.  Some clusters work, one does not.  I've attempted disconnecting & reconnecting, restarting the computer, removing the entry from .kube/config and regenerating.

**Comment by robbesettlemint** [comment]
Yes, even uninstalled and reinstalled Lens and resetted my kubeconfig.

**Comment by Nokel81** [comment]
Okay thanks, can you try turning off the "download kubectl" preference and then completely restarting lens (including tray) and seeing if that resolves the issue?

**Comment by robbesettlemint** [comment]
Tried that, but still having the same problem.

**Comment by Nokel81** [comment]
if you open up dev tools do you see any errors in the console?

**Comment by themooer1** [comment]
I'm having the same problem with the single cluster in `~/.kube/config`. Here is my kubeconfig (which worked on older versions of Lens) with minimal redactions:

```
apiVersion: v1
kind: Config
clusters:
  - name: kanaan
    cluster:
      server: https://joshua:6443
      certificate-authority-data: >-
        LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUJkekNDQVIyZ0F3SUJBZ0lCQURBS0JnZ3Foa2pPUFFRREFqQWpNU0V3SHdZRFZRUUREQmhyTTNNdGMyVnkKZG1WeUxXTmhRREUyTWpRek16Y3pORFF3SGhjTk1qRXdOakl5TURRME9UQTBXaGNOTXpFd05qSXdNRFEwT1RBMApXakFqTVNFd0h3WURWUVFEREJock0zTXRjMlZ5ZG1WeUxXTmhRREUyTWpRek16Y3pORFF3V1RBVEJnY3Foa2pPClBRSUJCZ2dxaGtqT1BRTUJCd05DQUFRZWJOMXBYejhVZWR2dzl3cDBPYkRXaHdla2pkQVNoR2gxTk0zSFg5blgKRU9VR0lGRUpnZjYrdXp0RU1MNG1zd3dOaElRdjlIbHJlc3RuMjg0b3dQMXNvMEl3UURBT0JnTlZIUThCQWY4RQpCQU1DQXFRd0R3WURWUjBUQVFIL0JBVXdBd0VCL3pBZEJnTlZIUTRFRmdRVUZVT1d1a1RBc3g5S1FBT21Dd052ClJpTkVyVW93Q2dZSUtvWkl6ajBFQXdJRFNBQXdSUUloQVA5cWRmd3I1QmFoQ29RcTVjaEF6bElubldNMitzWlYKMEpTTnc5SGFLMERwQWlBdERHajl3T0xNdkdmaDJwMUZDZCsrKzRCazRheXhmN1I4ZUxOS3ZCWnVhQT09Ci0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0K
      insecure-skip-tls-verify: false
users:
  - name: default
    user:
      client-certificate-data: >-
        LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUJrRENDQVRlZ0F3SUJBZ0lJZGhrcXR4eG5JSWN3Q2dZSUtvWkl6ajBFQXdJd0l6RWhNQjhHQTFVRUF3d1kKYXpOekxXTnNhV1Z1ZEMxallVQXhOakkwTXpNM016UTBNQjRYRFRJeE1EWXlNakEwTkRrd05Gb1hEVEl5TURZeQpNakEwTkRrd05Gb3dNREVYTUJVR0ExVUVDaE1PYzNsemRHVnRPbTFoYzNSbGNuTXhGVEFUQmdOVkJBTVRESE41CmMzUmxiVHBoWkcxcGJqQlpNQk1HQnlxR1NNNDlBZ0VHQ0NxR1NNNDlBd0VIQTBJQUJOOW5zSWZzNk1UeUhoK08KU05IS2UzOUEwRGFwZTZvNzF2TXNUOVJHeFdJR1hMSDA3d2ZwQ1RPVFBjQkJwRTd2cTE3amk4djFteThUMVhPVQpLTmY3MktpalNEQkdNQTRHQTFVZER3RUIvd1FFQXdJRm9EQVRCZ05WSFNVRUREQUtCZ2dyQmdFRkJRY0RBakFmCkJnTlZIU01FR0RBV2dCUmtQNEZ2aXZ6QnlmQVFYQ1Z4NTg3YWlLSitCekFLQmdncWhrak9QUVFEQWdOSEFEQkUKQWlCOXFWRER1am1heDNMYVFyUDV6aVlaWFV1YjR6cWg4ajFIdXZQTGhudkl5Z0lnSGFJb0UrYXVFNWxmMVdZUApNNTdLTVhqNjlteW5WMWZDd2ZsdHJ3TXdiWDg9Ci0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0KLS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUJkekNDQVIyZ0F3SUJBZ0lCQURBS0JnZ3Foa2pPUFFRREFqQWpNU0V3SHdZRFZRUUREQmhyTTNNdFkyeHAKWlc1MExXTmhRREUyTWpRek16Y3pORFF3SGhjTk1qRXdOakl5TURRME9UQTBXaGNOTXpFd05qSXdNRFEwT1RBMApXakFqTVNFd0h3WURWUVFEREJock0zTXRZMnhwWlc1MExXTmhRREUyTWpRek16Y3pORFF3V1RBVEJnY3Foa2pPClBRSUJCZ2dxaGtqT1BRTUJCd05DQUFRQkJqNFEza2g5Q3VOTk5RUmdKK1dla05LdHY5UGZHdjJrRSswTTloTlAKdmNTSEpQdE9RaHExRXZhZ0k1YkNaQ2VDdXJYV2swR1RWY0wva0hHSkxpYStvMEl3UURBT0JnTlZIUThCQWY4RQpCQU1DQXFRd0R3WURWUjBUQVFIL0JBVXdBd0VCL3pBZEJnTlZIUTRFRmdRVVpEK0JiNHI4d2Nud0VGd2xjZWZPCjJvaWlmZ2N3Q2dZSUtvWkl6ajBFQXdJRFNBQXdSUUloQUpSL3RScStLa2dJcjlFcFlMZktWa3hEbkVmNlBZQ2MKZlp0VW1CQ1d0eGZqQWlBeDUzSm8rVGVWeWhRS285ZXlFWEswanQvYjBUQTBZQzcvSXdPZWc1OXA4UT09Ci0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0K
      client-key-data: >-
        ABCDABCDABCD=
contexts: []
preferences: {}
current-context: kanaan
```

**Comment by themooer1** [comment]
This config works for the same cluster (same client key):

```
apiVersion: v1
clusters:
- cluster:
    certificate-authority-data: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUJkekNDQVIyZ0F3SUJBZ0lCQURBS0JnZ3Foa2pPUFFRREFqQWpNU0V3SHdZRFZRUUREQmhyTTNNdGMyVnkKZG1WeUxXTmhRREUyTWpRek16Y3pORFF3SGhjTk1qRXdOakl5TURRME9UQTBXaGNOTXpFd05qSXdNRFEwT1RBMApXakFqTVNFd0h3WURWUVFEREJock0zTXRjMlZ5ZG1WeUxXTmhRREUyTWpRek16Y3pORFF3V1RBVEJnY3Foa2pPClBRSUJCZ2dxaGtqT1BRTUJCd05DQUFRZWJOMXBYejhVZWR2dzl3cDBPYkRXaHdla2pkQVNoR2gxTk0zSFg5blgKRU9VR0lGRUpnZjYrdXp0RU1MNG1zd3dOaElRdjlIbHJlc3RuMjg0b3dQMXNvMEl3UURBT0JnTlZIUThCQWY4RQpCQU1DQXFRd0R3WURWUjBUQVFIL0JBVXdBd0VCL3pBZEJnTlZIUTRFRmdRVUZVT1d1a1RBc3g5S1FBT21Dd052ClJpTkVyVW93Q2dZSUtvWkl6ajBFQXdJRFNBQXdSUUloQVA5cWRmd3I1QmFoQ29RcTVjaEF6bElubldNMitzWlYKMEpTTnc5SGFLMERwQWlBdERHajl3T0xNdkdmaDJwMUZDZCsrKzRCazRheXhmN1I4ZUxOS3ZCWnVhQT09Ci0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0K
    server: https://joshua:6443
  name: default
contexts:
- context:
    cluster: default
    user: default
  name: default
current-context: default
kind: Config
preferences: {}
users:
- name: default
  user:
    client-certificate-data: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUJrRENDQVRlZ0F3SUJBZ0lJZGhrcXR4eG5JSWN3Q2dZSUtvWkl6ajBFQXdJd0l6RWhNQjhHQTFVRUF3d1kKYXpOekxXTnNhV1Z1ZEMxallVQXhOakkwTXpNM016UTBNQjRYRFRJeE1EWXlNakEwTkRrd05Gb1hEVEl5TURZeQpNakEwTkRrd05Gb3dNREVYTUJVR0ExVUVDaE1PYzNsemRHVnRPbTFoYzNSbGNuTXhGVEFUQmdOVkJBTVRESE41CmMzUmxiVHBoWkcxcGJqQlpNQk1HQnlxR1NNNDlBZ0VHQ0NxR1NNNDlBd0VIQTBJQUJOOW5zSWZzNk1UeUhoK08KU05IS2UzOUEwRGFwZTZvNzF2TXNUOVJHeFdJR1hMSDA3d2ZwQ1RPVFBjQkJwRTd2cTE3amk4djFteThUMVhPVQpLTmY3MktpalNEQkdNQTRHQTFVZER3RUIvd1FFQXdJRm9EQVRCZ05WSFNVRUREQUtCZ2dyQmdFRkJRY0RBakFmCkJnTlZIU01FR0RBV2dCUmtQNEZ2aXZ6QnlmQVFYQ1Z4NTg3YWlLSitCekFLQmdncWhrak9QUVFEQWdOSEFEQkUKQWlCOXFWRER1am1heDNMYVFyUDV6aVlaWFV1YjR6cWg4ajFIdXZQTGhudkl5Z0lnSGFJb0UrYXVFNWxmMVdZUApNNTdLTVhqNjlteW5WMWZDd2ZsdHJ3TXdiWDg9Ci0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0KLS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUJkekNDQVIyZ0F3SUJBZ0lCQURBS0JnZ3Foa2pPUFFRREFqQWpNU0V3SHdZRFZRUUREQmhyTTNNdFkyeHAKWlc1MExXTmhRREUyTWpRek16Y3pORFF3SGhjTk1qRXdOakl5TURRME9UQTBXaGNOTXpFd05qSXdNRFEwT1RBMApXakFqTVNFd0h3WURWUVFEREJock0zTXRZMnhwWlc1MExXTmhRREUyTWpRek16Y3pORFF3V1RBVEJnY3Foa2pPClBRSUJCZ2dxaGtqT1BRTUJCd05DQUFRQkJqNFEza2g5Q3VOTk5RUmdKK1dla05LdHY5UGZHdjJrRSswTTloTlAKdmNTSEpQdE9RaHExRXZhZ0k1YkNaQ2VDdXJYV2swR1RWY0wva0hHSkxpYStvMEl3UURBT0JnTlZIUThCQWY4RQpCQU1DQXFRd0R3WURWUjBUQVFIL0JBVXdBd0VCL3pBZEJnTlZIUTRFRmdRVVpEK0JiNHI4d2Nud0VGd2xjZWZPCjJvaWlmZ2N3Q2dZSUtvWkl6ajBFQXdJRFNBQXdSUUloQUpSL3RScStLa2dJcjlFcFlMZktWa3hEbkVmNlBZQ2MKZlp0VW1CQ1d0eGZqQWlBeDUzSm8rVGVWeWhRS285ZXlFWEswanQvYjBUQTBZQzcvSXdPZWc1OXA4UT09Ci0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0K
    client-key-data: ABCDABCDABCD=
```

**Comment by mbreuker** [comment]
Had the same problem. Creating additional config files, and then loading the cluster from the new config file solved the issue for me.

**Comment by Nokel81** [comment]
> I'm having the same problem with the single cluster in `~/.kube/config`. Here is my kubeconfig (which worked on older versions of Lens) with minimal redactions:
> 
> ```
> apiVersion: v1
> kind: Config
> clusters:
>   - name: kanaan
>     cluster:
>       server: https://joshua:6443
>       certificate-authority-data: >-
>         LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUJkekNDQVIyZ0F3SUJBZ0lCQURBS0JnZ3Foa2pPUFFRREFqQWpNU0V3SHdZRFZRUUREQmhyTTNNdGMyVnkKZG1WeUxXTmhRREUyTWpRek16Y3pORFF3SGhjTk1qRXdOakl5TURRME9UQTBXaGNOTXpFd05qSXdNRFEwT1RBMApXakFqTVNFd0h3WURWUVFEREJock0zTXRjMlZ5ZG1WeUxXTmhRREUyTWpRek16Y3pORFF3V1RBVEJnY3Foa2pPClBRSUJCZ2dxaGtqT1BRTUJCd05DQUFRZWJOMXBYejhVZWR2dzl3cDBPYkRXaHdla2pkQVNoR2gxTk0zSFg5blgKRU9VR0lGRUpnZjYrdXp0RU1MNG1zd3dOaElRdjlIbHJlc3RuMjg0b3dQMXNvMEl3UURBT0JnTlZIUThCQWY4RQpCQU1DQXFRd0R3WURWUjBUQVFIL0JBVXdBd0VCL3pBZEJnTlZIUTRFRmdRVUZVT1d1a1RBc3g5S1FBT21Dd052ClJpTkVyVW93Q2dZSUtvWkl6ajBFQXdJRFNBQXdSUUloQVA5cWRmd3I1QmFoQ29RcTVjaEF6bElubldNMitzWlYKMEpTTnc5SGFLMERwQWlBdERHajl3T0xNdkdmaDJwMUZDZCsrKzRCazRheXhmN1I4ZUxOS3ZCWnVhQT09Ci0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0K
>       insecure-skip-tls-verify: false
> users:
>   - name: default
>     user:
>       client-certificate-data: >-
>         LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUJrRENDQVRlZ0F3SUJBZ0lJZGhrcXR4eG5JSWN3Q2dZSUtvWkl6ajBFQXdJd0l6RWhNQjhHQTFVRUF3d1kKYXpOekxXTnNhV1Z1ZEMxallVQXhOakkwTXpNM016UTBNQjRYRFRJeE1EWXlNakEwTkRrd05Gb1hEVEl5TURZeQpNakEwTkRrd05Gb3dNREVYTUJVR0ExVUVDaE1PYzNsemRHVnRPbTFoYzNSbGNuTXhGVEFUQmdOVkJBTVRESE41CmMzUmxiVHBoWkcxcGJqQlpNQk1HQnlxR1NNNDlBZ0VHQ0NxR1NNNDlBd0VIQTBJQUJOOW5zSWZzNk1UeUhoK08KU05IS2UzOUEwRGFwZTZvNzF2TXNUOVJHeFdJR1hMSDA3d2ZwQ1RPVFBjQkJwRTd2cTE3amk4djFteThUMVhPVQpLTmY3MktpalNEQkdNQTRHQTFVZER3RUIvd1FFQXdJRm9EQVRCZ05WSFNVRUREQUtCZ2dyQmdFRkJRY0RBakFmCkJnTlZIU01FR0RBV2dCUmtQNEZ2aXZ6QnlmQVFYQ1Z4NTg3YWlLSitCekFLQmdncWhrak9QUVFEQWdOSEFEQkUKQWlCOXFWRER1am1heDNMYVFyUDV6aVlaWFV1YjR6cWg4ajFIdXZQTGhudkl5Z0lnSGFJb0UrYXVFNWxmMVdZUApNNTdLTVhqNjlteW5WMWZDd2ZsdHJ3TXdiWDg9Ci0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0KLS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUJkekNDQVIyZ0F3SUJBZ0lCQURBS0JnZ3Foa2pPUFFRREFqQWpNU0V3SHdZRFZRUUREQmhyTTNNdFkyeHAKWlc1MExXTmhRREUyTWpRek16Y3pORFF3SGhjTk1qRXdOakl5TURRME9UQTBXaGNOTXpFd05qSXdNRFEwT1RBMApXakFqTVNFd0h3WURWUVFEREJock0zTXRZMnhwWlc1MExXTmhRREUyTWpRek16Y3pORFF3V1RBVEJnY3Foa2pPClBRSUJCZ2dxaGtqT1BRTUJCd05DQUFRQkJqNFEza2g5Q3VOTk5RUmdKK1dla05LdHY5UGZHdjJrRSswTTloTlAKdmNTSEpQdE9RaHExRXZhZ0k1YkNaQ2VDdXJYV2swR1RWY0wva0hHSkxpYStvMEl3UURBT0JnTlZIUThCQWY4RQpCQU1DQXFRd0R3WURWUjBUQVFIL0JBVXdBd0VCL3pBZEJnTlZIUTRFRmdRVVpEK0JiNHI4d2Nud0VGd2xjZWZPCjJvaWlmZ2N3Q2dZSUtvWkl6ajBFQXdJRFNBQXdSUUloQUpSL3RScStLa2dJcjlFcFlMZktWa3hEbkVmNlBZQ2MKZlp0VW1CQ1d0eGZqQWlBeDUzSm8rVGVWeWhRS285ZXlFWEswanQvYjBUQTBZQzcvSXdPZWc1OXA4UT09Ci0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0K
>       client-key-data: >-
>         ABCDABCDABCD=
> contexts: []
> preferences: {}
> current-context: kanaan
> ```

The reason that this config doesn't make a cluster show up in lens is because there is not contexts defined

**Comment by brainstain** [comment]
> Had the same problem. Creating additional config files, and then loading the cluster from the new config file solved the issue for me.

This worked for me as well.  I copy/pasted the data out of the original file, which makes this very strange.

**Comment by willsilvano** [comment]
Same problem here.

**Comment by makowskimateusz** [comment]
For me coping "config" over to "config2" and syncing this file fixed the issue
<img width="823" alt="Screenshot 2021-09-30 at 11 22 42" src="https://user-images.githubusercontent.com/5725006/135425769-5ef42308-81e7-4925-a9a2-787c070f5dd0.png">

**Comment by serhiiromaniuk** [comment]
Hello!

Latest release for now 5.2.4 does not working, `kubectl --kubeconfig ...` -- OK. 
Tested latest releases on MacOs, Ubuntu 21 and Windows

**Comment by Nokel81** [comment]
@serhiiromaniuk By not working you mean that the when you click on the cluster to open it you get the grey screen as above?

**Comment by serhiiromaniuk** [comment]
@Nokel81 Yeah, sure. Sorry for inaccuracies.
Like, when you are selecting OR connecting in catalog, connecting in pin bar -- not grey screen, but disappearing.
Sourcing in different directories, `~/...`, `~/.kube/...`

**Comment by Nokel81** [comment]
Sorry, but I don't understand. Are you clusters there but if you try and connect it doesn't show anything or is it that they don't exist in the catalog in the first place?

**Comment by rngcntr** [comment]
I think I am facing the same issue. I recently added a new context to one of my clusters, which I was able to access yesterday. Today, when I try to connect to the context, the text "Connecting... Authentication proxy started" is displayed, and once the connection is established, the window simply turns grey. All other contexts on the same cluster are still working as expected.

This image shows the behavior for me: As you can see, the context "TPA" is connected (indicated by the blue dot) and selected (indicated by the white frame) but nothing is displayed:
![image](https://user-images.githubusercontent.com/7890887/135591557-e4ab15b0-66e4-4371-b588-c9bed9b04ec7.png)

**Comment by serhiiromaniuk** [comment]
Hello there!
As a "hotfix", I offer you guys install `v4.2.4` https://github.com/lensapp/lens/releases/tag/v4.2.4

Seems to be enough stable for a cluster observation and common tasks.

**Comment by serhiiromaniuk** [comment]
> Sorry, but I don't understand. Are you clusters there but if you try and connect it doesn't show anything or is it that they don't exist in the catalog in the first place?

@Nokel81 My clusters and configs are OK. Like everything appears in Catalog. But when I just clicking to Connect in the catalog OR on Pinbar it disappears. 100% same as on @rngcntr screenshot above.

**Comment by Nokel81** [comment]
Can you post some renderer logs by opening up the devtools and posting a screenshot of the console @serhiiromaniuk ? Many thanks.

**Comment by rngcntr** [comment]
![image](https://user-images.githubusercontent.com/7890887/135812091-326e7a58-fd1f-49aa-b40f-ca517f4f98d8.png)

I opened lens, connected to a working context and switched to one of the broken contexts. The top of the log is from the working one, bottom is from the broken one.

EDIT: One more thing I noticed: The lens renderer causes 100% CPU usage while trying to load the broken context:
![image](https://user-images.githubusercontent.com/7890887/135813340-68b37720-811f-4188-a469-d8d6e203ab80.png)

After minutes, Lens pauses itself because of a potential OOM:
![image](https://user-images.githubusercontent.com/7890887/135813654-9f308f21-4111-4b2a-a284-bf7225b721f2.png)

**Comment by paulochf** [comment]
> Had the same problem. Creating additional config files, and then loading the cluster from the new config file solved the issue for me.

I'm here to say this worked, as like with @brainstain and @makowskimateusz .

My app versions are
```
Lens: 5.2.5-latest.20211001.2
Electron: 12.2.1
Chrome: 89.0.4389.128
Node: 14.16.0
```

Update: ... and then stopped working again.

**Comment by makowskimateusz** [comment]
Because this issue is quite annoying on recent Lens version I discovered another tool. I guess for all of you usingIntelij Idea plugin: https://plugins.jetbrains.com/plugin/10485-kubernetes will be more handy.

**Comment by acedward** [comment]
Duplicate works.
`cp .kube/config .kube/config2` and in Lenz Preferences > Kubernets > Sync Files and Folders, shows the "duplicated" clusters that work.  Lenz 5.2.5-latest.20211001.2 @ Pop OS 21.04

**Comment by rngcntr** [comment]
Duplicate had no effect for me

**Comment by Saeger** [comment]
I also confirm that duplicating my config file worked for me and I was able to see the K8s cluster resources again.

**Comment by thomastvedt** [comment]
Duplicate worked..
It worked the "first time" without duplicating

**Comment by like-a-bause** [comment]
Another potential workaround: I started deleting stuff in ~/Library/Application Support/Lens/ (Mac OS).
After deleting the contents of ~/Library/Application Support/Lens/lens-local-storage/ the cluster showed up again.
As I deleted a bunch of other stuff (Cache, Local Storage, Session Storage, database, kubeconfigs) before that I can't be 100% sure if it's enough to just delete the lens-local-storage files.
Maybe someone can try this out. I will confirm once the bug strikes me again.
Would be a better approach than copying your kubecfg around again and again.

**Comment by joaocc** [comment]
Removing ~/Library/Application Support/Lens/lens-local-storage/ also solved it for me (#4125)

**Comment by Nokel81** [comment]
I am going to close this as a duplicate of #4018 because I think it is the same issue. Will still use all the info here to help with fixing the issue.

**Comment by zot11** [comment]
cp ~/.kube/config ~/lens-conf 
and add cluster from lens-conf in UI
- works for me.

**Comment by ErezWeiss** [comment]
Still here

**Comment by sasagarw** [comment]
Facing same issue still. Any other solution?
Open Lens version - latest (v6.3.0)

**Comment by Nokel81** [comment]
@sasagarw There was a bug in 6.3.0 which was fixed by https://github.com/lensapp/lens/pull/6835 and will be part of an upcoming release

**Comment by NKJensen** [comment]
Hi all - on windows installed via "choco install openlens", I had the same issue. I fixed it by deleting the appdata settings:

%appdata%/OpenLens

and restarting OpenLens app. The extension was preserved - it is stored in %userprofile%/.k8lens/extensions

**Comment by anton-martynov** [comment]
> Removing ~/Library/Application Support/Lens/lens-local-storage/ also solved it for me (#4125)

also works in Lens 2023.9, Windows 10

**Comment by sileilie** [comment]
same here, I've deleted %appdata%/OpenLens/cache and local storage and then it worked
